### PR TITLE
[Merged by Bors] - feat: port MeasureTheory.Integral.Bochner

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by int-y1** [comment]
33 messages left

**Comment by mcdoll** [comment]
There are two lemmas with errors and the `integral_simps` attribute is not yet working.

**Comment by urkud** [comment]
I moved attr declaration to a new file.

**Comment by urkud** [comment]
@Komyyy Are you working on this branch now?

**Comment by Komyyy** [comment]
@urkud Yes.

**Comment by Komyyy** [comment]
Except `integral_simps`, the work is done.
@urkud Can you push the commit which move attr?

**Comment by urkud** [comment]
@Ruben-VandeVelde CI passes. Do you want to have a look before we change it to `awaiting-review`?

File: Mathlib/MeasureTheory/Integral/Bochner.lean [hunk 1]
```diff
@@ -515,87 +515,97 @@ and prove basic properties of this integral. -/
...
     integral f = integral g :=
   SimpleFunc.integral_congr (SimpleFunc.integrable f) h
 #align measure_theory.L1.simple_func.integral_congr MeasureTheory.L1.SimpleFunc.integral_congr
 
 theorem integral_add (f g : α →₁ₛ[μ] E) : integral (f + g) = integral f + integral g :=
-  setToL1s_add _ (fun _ _ => weightedSmul_null) weightedSmul_union _ _
+  setToL1S_add _ (fun _ _ => weightedSMul_null) weightedSMul_union _ _
 #align measure_theory.L1.simple_func.integral_add MeasureTheory.L1.SimpleFunc.integral_add
 
+-- Porting note: why did this need to be added?
```
Comment (jcommelin) [inline Mathlib/MeasureTheory/Integral/Bochner.lean:545]: @semorrison Do you recognize this as related to an existing issue?

**Comment by jcommelin** [comment]
There is one instance that isn't found automatically. It is flagged by a porting note.
All the other porting notes seem fairly normal to me. So I think this is basically mergeable. We might want to dig into that instance problem though.

File: Mathlib/MeasureTheory/Integral/Bochner.lean (hunk 1 above)
Comment (semorrison) [reply Mathlib/MeasureTheory/Integral/Bochner.lean:545]: No, I'm mystified. The first of the three haveI's is unnecessary, so I'll remove that, at least.

**Comment by semorrison** [comment]
Oops, sorry, didn't see you'd merge. I just added some more information about the shortcut instance.

bors merge

**Comment by Ruben-VandeVelde** [comment]
Thanks all!

**Pull request closed without merging**